src/main/java/com/virtudoc/web/entity/UserEntity.java

```diff
@@ -0,0 +1,159 @@
+package com.virtudoc.web.entity;
+
+import org.springframework.format.annotation.DateTimeFormat;
+
+import javax.persistence.*;
+import javax.validation.constraints.NotEmpty;
+import javax.validation.constraints.NotNull;
+import java.util.Date;
+import java.util.Set;
+
+@Entity
+@Table(name = "user")
+public class UserEntity {
```

Comment:
Just as a note, the database layer for user accounts was already implemented in #25. You'll have to merge in the `main` branch to see it. The entity class name is `UserAccount`.